mm2src/mm2_bin_lib/Cargo.toml

```diff
@@ -5,8 +5,8 @@
 
 [package]
 name = "mm2_bin_lib"
-version = "2.1.0-beta"
-authors = ["James Lee", "Artem Pikulin", "Artem Grinblat", "Omar S.", "Onur Ozkan", "Alina Sharon", "Caglar Kaya", "Cipi", "Sergey Boiko", "Samuel Onoja", "Roman Sztergbaum", "Kadan Stadelmann <ca333@komodoplatform.com>", "Dimxy"]
+version = "2.2.0-beta"
+authors = ["James Lee", "Artem Pikulin", "Artem Grinblat", "Omar S.", "Onur Ozkan", "Alina Sharon", "Caglar Kaya", "Cipi", "Sergey Boiko", "Samuel Onoja", "Roman Sztergbaum", "Kadan Stadelmann <ca333@komodoplatform.com>", "Dimxy", "mariocynicys"]
```

Comment:
@mariocynicys please check if this is the name you want in authors.

Reply:
prefer `Omer Yacine` here for this one.